src/main/java/capstone/mfslbackend/response/dto/TeamsResponse.java

```diff
@@ -13,4 +13,25 @@
 @ToString
 public class TeamsResponse {
     TeamResponse team;
+    TeamResponse home;
```

Comment:
@Data does all the getters and setters.
@AllArgsConstructor gives you that constructor